scripts/styleChecker.fsx

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env -S dotnet fsi
+
+#r "nuget: Fsdk, Version=0.6.0--date20230326-0544.git-5c4f55b"
+
+open Fsdk
+open Fsdk.Process
+
+let StyleFSharpFiles() =
+    Process
+        .Execute(
+            {
+                Command = "dotnet"
+                Arguments = "new tool-manifest --force"
+            },
+            Process.Echo.Off
+        )
+        .UnwrapDefault()
+    |> ignore
+
+    Process
+        .Execute(
+            {
+                Command = "dotnet"
+                Arguments =
+                    "tool install fantomless-tool --version 4.7.997-prerelease"
+            },
+            Process.Echo.Off
+        )
+        .UnwrapDefault()
+    |> ignore
+
+    Process
+        .Execute(
+            {
+                Command = "dotnet"
+                Arguments = "fantomless --recurse ."
+            },
+            Process.Echo.Off
+        )
+        .UnwrapDefault()
+    |> ignore
+
+let RunPrettier(arguments: string) =
+    let processResult =
+        Process
+            .Execute(
+                {
+                    Command = "npx"
+                    Arguments = $"prettier {arguments}"
+                },
+                Process.Echo.All
+            )
+            .Result
+
+    match processResult with
+    | ProcessResultState.Success output -> output
+    | ProcessResultState.WarningsOrAmbiguous output -> output.StdErr
```

Comment:
@realmarv in this case you're printing StdErr but not StdOut

Reply:
Done